framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileResource.java

```diff
@@ -3,7 +3,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 
-/** Contract for sources of stub or ajava data. */
+/** Interface for annotation files (stub files or ajava files). */
```

Comment:
The original "source" terminology was correct, because instances of this interface don't contain the ajava file data themselves, they contain the information for how to locate the file (e.g. `File` object or jar file entry).